tutorials/cyber/raw_data_generator/.gitignore

```diff
@@ -0,0 +1 @@
+test/data/output/*
```

Comment:
This looks like an accidental inclusion

Reply:
The result of the unit test `test_raw_data_generator.py` outputs a file here and I didn't initially want to include it in the repo. Should I add it back in?

Reply:
This has been removed and is writing to `/data/lanl/output` directory